src/main/java/com/example/bank/account/TransferType.java

```diff
@@ -0,0 +1,8 @@
+package com.example.bank.account;
+
+public enum TransferType {
+    DOMESTIC,
+    OWN_TRANSFER,
+    TAX_CUSTOMS,
+    ZUS
```

Comment:
jak masz zus to i krus powinien byc. 
PYtanie czy w ogole sens to robic. Bo ZUS nikomu z poza polski nic nie powie